llvm/test/Instrumentation/SanitizerCoverage/coverage-dbg.ll

```diff
@@ -16,7 +16,7 @@
 
 ; Test that __sanitizer_cov_trace_pc_guard call has !dbg pointing to the opening { of A::f().
 ; CHECK: call void @__sanitizer_cov_trace_pc_guard(ptr{{.*}}) #{{.*}}, !dbg [[A:!.*]]
```

Comment:
This comment will want updating as it's now not true I guess; it's unclear to me whether this indicates there are expectations that the coverage call gets a "real" source location?